### DISCUSSION TITLE
Welcome to UE_TAPython_Plugin_Release Discussions!

### DISCUSSION THREAD

**Discussion by cgerchenhp** [cgerchenhp/UE_TAPython_Plugin_Release#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DenCommander** [comment]
Hello. When I rebuild my unreal engine project, that uses this plugin, the Unrealeditor-Tapython.dll and unrealeditor-tapython.pdb files are deleted from the Plugins\TAPython\Binaries\Win64 folder and the project does not start, because plugin not compiled. I have to unzip the plugin into the project folder again (or download from perforce). As I understand it, you don't include *.cpp and *.h source files in the archive, so the plugin cannot be compiled. How can this issue be fixed?

**Reply by cgerchenhp** [reply]
Actually, I don't have a one-size-fits-all solution to this problem at the moment, but here are a few feasible methods:

- Try using the 'Build' command instead of the 'Rebuild' command in Visual Studio. Rebuild removes the .dll and .pdb files, while Build does not.
- If you rename the extension of TAPython.uplugin to something else before performing a Rebuild, it can also prevent it from being deleted. However, you'll need to restore the original extension once the compilation is complete.
- Back up the .dll and .pdb files in the win64 directory and create a .bat file to restore them after the rebuild.
- You can use the command prompt to rebuild the project or utilize other automatic build functionalities, incorporating a step to restore the .dll files in the process.

If there are better solutions in the future, I will post them here. Thank you for your feedback.
